providers/implementations/kdfs/krb5kdf.c

```diff
@@ -0,0 +1,434 @@
+/*
+ * Copyright 2018-2019 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdlib.h>
+#include <stdarg.h>
+#include <string.h>
+
+#include <openssl/core_names.h>
+#ifndef OPNESSL_NO_DES
```

Comment:
There is a typo OPNESSL->OPENSSL. Although this ifndef is not needed for no-des compilability because the header file is there and contains OPENSSL_NO_DES guards internally.

Reply:
Sigh I already had corrected this same typo elsewhere, but I guess the internal check masked this one...